tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -569,12 +570,13 @@ func TestRateLimit(t *testing.T) {
 	got, err = vectorValue(value, map[string]string{responseCodeLabel: "200", "destination_version": "v1"})
 	if err != nil {
 		t.Logf("prometheus values for request_count:\n%s", promDump(promAPI, "request_count"))
-		fatalf(t, "Could not find successes value: %v", err)
+		t.Errorf("Could not find successes value: %v", err)
+		got = 0
 	}
 
 	got = got - prior200s
```

Comment:
Note: this and the above may get negative if somehow the first call works and the second one doesn't but it's still nice for the normal case (only 200s) we've had many times when things broke